about.html

```diff
@@ -0,0 +1,122 @@
+                      <a href="#">Sponsor</a>
+                    </li>
+                    <li class="links">
+                      <a href="#">News</a>
+                    </li>
+                    <li class="highlighter"><a class="high" href="">Queens Gambit</a></li>
+                  </ul>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="head-liner">
+            <div class="headliner">
+            <p class="hello">Hello Chess players!</p>
+            <h1>Global Chess Tournament</h1>
+            <hr>
+            <p class="about-focus">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dignissim eros ac justo feugiat interdum. Morbi sit amet tortor vitae nunc facilisis efficitur. Etiam tincidunt mollis ultrices. Nam a tincidunt augue. Vestibulum interdum massa dignissim tortor porta hendrerit. Donec efficitur molestie massa eu eleifend. Phasellus mattis, arcu non iaculis mollis, enim metus consequat ex, nec volutpat nibh arcu vel mi.</p>
+            <p class="contact">Please contact us by email for any further questions about the CHESS Tournament 2022</p>
+            <p class="email">admin@chess.com</p>
+        </div>
+        </section>
+        <section class="tourna-ment">
+            <div class="tournament">
+            <h2>CHESS 2022 logo</h2>
+            <hr>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dignissim eros ac justo feugiat interdum. Morbi sit amet tortor vitae nunc facilisis efficitur. Etiam tincidunt mollis ultrices. Nam a tincidunt augue. </p>
+            <div class="big-div">
+            <i class="fa-solid fa-chess big-logo"></i>
+            </div>
+            </div>
+        </section>
+        <section>
+        <div class="past-events">
+            <h2>See Chess Tournament past events!</h2>
+            <hr>
+            <p class="event-desc">Duis tincidunt leo leo, non ultrices nisi malesuada sed. Nulla sagittis, urna vel lobortis tempus, elit felis sodales turpis, ac gravida dui libero efficitur enim.</p>
+            <div class="event-pics">
+                <div class="event-card1">
+                    <h4>2020</h4>
+                    <p>Charlotte Summer GM A 2022</p>
+                </div>
+                <div class="event-card2">
+                    <h4>2021</h4>
+                    <p>Chess.com CGC Play In Qualifiers 2022</p>
+                </div>
+            </div>
+        </div>
+        </section>
+        <section class="partners">
+            <h6>Partners</h6>
+            <hr>
+            <div>
+            <i class="fa-brands fa-airbnb partner"></i>
+            <i class="fa-brands fa-google partner"></i>
+            <i class="fa-brands fa-xbox partner"></i>
+            <i class="fa-brands fa-airbnb partner"></i>
+            <i class="fa-brands fa-google partner"></i>
+            <i class="fa-brands fa-xbox partner"></i>
+         </div>
+        </section>
```

Comment:
- [ ] Please consider using `ul` and `li` tags for the list of partners.

- [ ] Kindly fix the horizontal scroll that appears on the mobile version (width: 375px) on the "About" page. The horizontal scroll is caused by this section where each `i` tag has a margin-right. The last `i` tag overflows its margin right and causes the horizontal scroll.